testsuite/openmodelica/fmi/ModelExchange/2.0/testFmiInitialUnknowns03.mos

```diff
@@ -0,0 +1,99 @@
+// name:  testFmiInitialUnknowns03.mos
+// keywords: FMI 2.0 export
+// status: correct
+// teardown_command: rm -rf binaries sources modelDescription.xml InitialUnknown03_tmp.xml initialUnknownstest03.xml
+
+loadString("
+model testFmiInitialUnknowns03
+ input Real u;
+ Real x(fixed=true, start=0.0);
+ output Real y;
+equation
+ der(x) = u;
+ y = x;
+end testFmiInitialUnknowns03;
+");
+getErrorString();
+
+translateModelFMU(testFmiInitialUnknowns03);
+getErrorString();
+
+// unzip to console, quiet, extra quiet
+system("unzip -cqq testFmiInitialUnknowns03.fmu modelDescription.xml > InitialUnknown03_tmp.xml"); getErrorString();
+system("sed -n \"/<ModelVariables>/,/<\\/ModelVariables>/p\" InitialUnknown03_tmp.xml > initialUnknownstest03.xml"); getErrorString();
+readFile("initialUnknownstest03.xml"); getErrorString();
+
+system("sed -n \"/<ModelStructure>/,/<\\/ModelStructure>/p\" InitialUnknown03_tmp.xml > initialUnknownstest03.xml"); getErrorString();
+readFile("initialUnknownstest03.xml"); getErrorString();
+
+// Result:
+// true
+// ""
+// "testFmiInitialUnknowns03.fmu"
+// ""
+// 0
+// ""
+// 0
+// ""
+// "  <ModelVariables>
+//   <!-- Index of variable = \"1\" -->
+//   <ScalarVariable
+//     name=\"x\"
+//     valueReference=\"0\"
+//     variability=\"continuous\"
+//     causality=\"local\"
+//     initial=\"exact\">
+//     <Real start=\"0.0\"/>
+//   </ScalarVariable>
+//   <!-- Index of variable = \"2\" -->
+//   <ScalarVariable
+//     name=\"der(x)\"
+//     valueReference=\"1\"
+//     variability=\"continuous\"
+//     causality=\"local\"
+//     initial=\"calculated\">
+//     <Real derivative=\"1\"/>
+//   </ScalarVariable>
+//   <!-- Index of variable = \"3\" -->
+//   <ScalarVariable
+//     name=\"u\"
+//     valueReference=\"2\"
+//     variability=\"continuous\"
+//     causality=\"input\"
+//     >
+//     <Real start=\"0.0\"/>
+//   </ScalarVariable>
+//   <!-- Index of variable = \"4\" -->
+//   <ScalarVariable
+//     name=\"y\"
+//     valueReference=\"3\"
+//     variability=\"continuous\"
+//     causality=\"output\"
+//     initial=\"calculated\">
+//     <Real/>
+//   </ScalarVariable>
+//   <!-- Index of variable = \"5\" -->
+//   <ScalarVariable
+//     name=\"_D_outputAlias_y\"
+//     valueReference=\"0\"
+//     variability=\"continuous\"
+//     causality=\"local\"
```

Comment:
Why doesn't this variable have an `initial` attribute?

Reply:
because it is an alias variable and initial attribute is not generated for alias variables in current implementation

Reply:
I see! Are there special rules for variability/causality/initial of alias variables in the FMI specification?

Reply:
There are some special rules for variability which we already fixed in this ticket[ #5673 ](https://trac.openmodelica.org/OpenModelica/ticket/5673), But Not for Causality and Initial

Reply:
That means an alias variable should be present with all the attributes, i.e., variability/causality/initial?